fsharp-backend/tests/FuzzTests/DvalRepr.FuzzTests.fs

```diff
@@ -36,7 +36,6 @@ module DeveloperRepr =
       |> Arb.filter (function
         | RT.DFnVal _ -> false
         | RT.DFloat 0.0 -> false
-        | RT.DFloat infinity -> false // TODO I don't think this is doing what we want it to!!
```

Comment:
FYI this was excluding all DFloats from this DvalRepr test. I've implemented the intention in `Generators.fs` (along with a NaN exclusion)

Reply:
Oh lol!